### DISCUSSION TITLE
Recevoir le type de formulaire dans le JSON de la transmission

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#193] [Annonces]
Bonjour,

Pouvez-vous faire un ajustement pour recevoir le type de formulaire dans le JSON de transmission.

Merci,

**Comment by anisite** [comment]
Bonsoir @ChrisThe01, je ne veux pas paraître "téteux", mais est-ce possible de transférer la conversation en issue svp ? pour toute demande d'intervention pour un bogue ou une fonctionnalité il faut passer par l'onglet "issues", pour le moment il est possible de cliquer à droite sur :
![image](https://github.com/MTESSDev/FRW/assets/3194288/2e8c247e-1f6a-4faa-9dc1-e836f02528e1)

Cela vous permet de suivre l'état des demandes et voir à quel moment cela est livré en production. 

Merci

**Reply by anisite** [reply]
c'est bon l'issue a été ouverte ici https://github.com/MTESSDev/FRW/issues/194

**Reply by ChrisThe01** [reply]
Je ne suis pas capable de le transformé en Issue, j'ai le message suivant :
![image](https://github.com/MTESSDev/FRW/assets/129303869/504c922f-b150-4f90-bf89-6605f9ba139c)
